README.md

```diff
@@ -1191,6 +1192,41 @@ messagingSession.addObserver(observer);
 messagingSession.start();
 ```
 
+### Providing application metadata
+
+Amazon Chime SDK for JavaScript allows builders to provide application metadata in the meeting session configuration. This field is optional. Amazon Chime uses application metadata to analyze meeting health trends or identify common failures to improve your meeting experience. 
+
+> ⚠️ Do not pass any Personal Identifiable Information (PII).
+
+**Use case 35.** Provide application metadata to the meeting session configuration.
+
+```
+import { MeetingSessionConfiguration, ApplicationMetadata } from 'amazon-chime-sdk-js';
+
+const createMeetingResponse = // CreateMeeting API response.
+const createAttendeeResponse = // CreateAttendee API response.
+const meetingSessionConfiguration = new MeetingSessionConfiguration(
+  createMeetingResponse,
+  createAttendeeResponse
+);
+
+meetingSessionConfiguration.applicationMetadata = ApplicationMetadata.create({
+  appName: 'AppName',
+  appVersion: '1.0.0'
+});
+```
+
+#### Accepted application metadata constraints
+
+```
+// The appName must be between 1-32 characters and satisfy following regular expression: /^[a-zA-Z0-9]+[a-zA-Z0-9_-]*[a-zA-Z0-9]+$/g
```

Comment:
optional: You can break down this comment into two lines, so readers with a laptop screen do not need to scroll horizontally.

Reply:
Cool, makes sense! Updated it, thanks!